app/controllers/avo/base_controller.rb

```diff
@@ -325,7 +325,7 @@ def set_index_params
         end
       end
 
-      @index_params[:sort_direction] = params[:sort_direction] || :desc
+      @index_params[:sort_direction] = params[:sort_direction] || @resource.default_sort_direction || :desc
```

Comment:
Since `class_attribute :default_sort_direction, default: :desc` default is `:desc` the `|| :desc` here is redundant.

```suggestion
      @index_params[:sort_direction] = params[:sort_direction] || @resource.default_sort_direction
```

Reply:
Good spot!